packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -54,10 +54,15 @@ export abstract class EncryptionKeyRef extends Construct {
 
     /**
      * Adds a statement to the KMS key resource policy.
+     * @param statement The policy statement to add
+     * @param allowNoOp If this is set to `false` and there is no policy
+     * defined (i.e. external key), the operation will fail. Otherwise, it will
```

Comment:
Align the text for the `@param`, or your Sphinx output will be all funked-up!

Reply:
What?! how is this related to my sphinx output?

Reply:
This is my sphinx output:

```rst
   .. py:method:: addToResourcePolicy(statement, [allowNoOp])

      Adds a statement to the KMS key resource policy.


      :param statement: The policy statement to add
      :type statement: :py:class:`@aws-cdk/cdk.PolicyStatement`
      :param allowNoOp: If this is set to `false` and there is no policy defined (i.e. external key), the operation will fail. Otherwise, it will no-op.
      :type allowNoOp: boolean or undefined
```

Looks fine.

Reply:
Weird. I had bunch of cases where this generated invalid `rst`. Maybe typescript changed how it parses jsdoc tags?